spec/iss67-spec.js

```diff
@@ -10,7 +10,9 @@ import {init as generateConfig} from '../commands/generate';
 
 describe('editorconfig', () => {
 	beforeEach(() => {
-		waitsForPromise(() => atom.packages.activatePackage('editorconfig'));
+		waitsForPromise(() => {
+			return atom.packages.activatePackage('editorconfig');
+		});
```

Comment:
Inline arrow functions implicitly returns. The previous and current has the exact same behavior.